libs/backend-libs/modules/account-management/src/lib/account/dto/query-account.dto.ts

```diff
@@ -1,9 +1,30 @@
-/**
- * classes implementing IAccountQueryOne and IAccountQueryMany
- */
-
-import { IAccountQueryOne, IAccountQueryMany } from './interfaces/query-account.interface';
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { IsNumber, IsOptional } from 'class-validator';
+import { Transform } from 'class-transformer';
+import { transformStringToBoolean } from '@backend-demo/shared/transformers';
+import {
+	IAccountQueryMany,
+	IAccountQueryOne,
+} from './interfaces/query-account.interface';
 
 export class QueryOneAccountDto implements IAccountQueryOne {}
 
-export class QueryManyAccountDto implements IAccountQueryMany {}
+export class QueryManyAccountDto implements IAccountQueryMany {
+	@ApiPropertyOptional({
+		type: Boolean,
+		default: null,
+	})
+	@IsOptional()
+	@Transform(transformStringToBoolean)
+	isActive?: boolean;
+
+	@ApiPropertyOptional({ example: 1 })
+	@IsNumber()
+	@IsOptional()
+	page?: number;
```

Comment:
Nejsem si jistý, ale tady asi budeš taky potřebovat transformaci stringu na number.

Reply:
V tomto novém PR je to opravené.
https://github.com/cesek-navisys/backend-demo/pull/32